content/talks/posters.bib

```diff
@@ -193,13 +193,6 @@ @inproceedings{Vogelstein2010e
 url = {http://dx.doi.org/10.6084/m9.figshare.1285819},
 year = {2010}
 }
-@inproceedings{Vogelstein2009a,
```

Comment:
you're mostly adding stuff -- why is this one deleted?

Reply:
It was a duplicate of another talk